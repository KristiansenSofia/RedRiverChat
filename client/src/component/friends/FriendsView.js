import React, { Component } from 'react';

import {friendsViewStyles} from "../../styles/FriendsStyles";
import '../../styles/Styles.css'

class FriendsView extends Component {
    render() {
        return (
            <div className="FriendsView">
                <p>Friends View</p>
            </div>

        );
    }
}

export default FriendsView;