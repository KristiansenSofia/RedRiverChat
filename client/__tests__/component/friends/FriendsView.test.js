import React from 'react';
import FriendsView from '../../../src/component/friends/FriendsView';
import renderer from 'react-test-renderer';

test('FriendsView renders without crashing', () => {
  const tree = renderer.create(
    <FriendsView state = {{ isSignedIn: true }}/>
  );
  expect(tree.toJSON()).toMatchSnapshot();
});
