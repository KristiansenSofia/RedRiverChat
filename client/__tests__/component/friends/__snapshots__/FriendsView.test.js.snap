// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FriendsView renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FriendsView
    state={
      Object {
        "isSignedIn": true,
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p>
        Friends View
      </p>,
      "className": "FriendsView",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Friends View",
      },
      "ref": null,
      "rendered": "Friends View",
      "type": "p",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          Friends View
        </p>,
        "className": "FriendsView",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Friends View",
        },
        "ref": null,
        "rendered": "Friends View",
        "type": "p",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
